validator/testcases/javascript/predefinedentities.py

```diff
@@ -45,6 +59,14 @@
         "All such changes must be made only via the browser search service, "
         "and only after an explicit opt-in from the user. All such changes "
         "must be reverted when the extension is disabled or uninstalled.",
+    "signing_help": (
+        "You should remove all references to these preferences from your "
+        "code, and interact with search settings only via the "
+        "`Services.search` interface. Extensions which interact with these "
+        "preferences directly will not be accepted for signing.",
```

Comment:
I guess I mean they won't be accepted at all, yeah. But "will not be accepted" doesn't feel right without a qualifier. I'll try to think of a different wording.